extension/CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [1.20]
 
+- Fixes:
+  - Fixed bug in `NAB: Refresh XLF files from g.xlf` where notes highlighting empty sources were not exported. Big thanks to [phenno1](https://github.com/phenno1) for reporting this in [issue 333](https://github.com/jwikman/nab-al-tools/issues/322).
+- New settings:
+  - `NAB.PreferLockedTranslations` Specifies if \"NAB: Refresh XLF files from g.xlf\" should be opinionated about locked translations e.g. when both source and target consists of only whitespace.
+
+## [1.19]
```

Comment:
What's going on here? :sweat_smile:

Reply:
Read  https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions for background info

My take in that is as follows:
- (almost) All new features are introduced in pre-releases
- I add the changelog for new features for the pre-release version
- When doing a release, I change the version header for the pre-release version to the release version
- In the end, there should be no pre-release versions (odd numbers) in the changelog for a release

In your case, this will be a part of the next pre-release, v1.21.
So add your stuff under a new header for v1.21. (there should be no v1.19 header, it's probably there because of merge conflicts...?)

Reply:
Ah, I had totally missed the changelog part :)